perf/memtx.cc

```diff
@@ -188,7 +188,7 @@ class Memtx final {
 		::user_cache_free();
 		::space_cache_destroy();
 
-		memtx->base.vtab->shutdown(&memtx->base);
+		memtx->base.vtab->free(&memtx->base);
```

Comment:
This change will require a patch for EE. Please prepare one.

Reply:
Done https://github.com/tarantool/tarantool-ee/pull/684.